### PR TITLE
Views

### PR DESCRIPTION
**Pull request by juanmanuelbyc**
Update views files to match the wireframes design. Update controllers files and style.

### CODE REVIEW

**Comment by omarsalem7** [comment]
Hi @juanmanuelbyc 😊, 

While you made a great effort in this project, I cannot proceed to review your code.

## Invalid Code Review Request 🛑 💀 

You have submitted a project, that has falling linters, kindly fix them and submit again

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by rloterh — COMMENTED** [review]
Hi Juan, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

Kindly ensure that provided placeholder texts/ guide instructions are removed and/or replaced with appropriate contents applicable under each section of your ReadMe. 

image

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: app/views/posts/index.html.erb [hunk 1]
```diff
@@ -1,3 +1,36 @@
...
+          <span class="likes">Likes:<%= post.likes_counter%></span>
+        </div>
+      </a>
+      
+      </div>
+      <div class="comments-container">
+        <% @comments=Post.find(post.id).comments %>
+        <% @comments.each do |comment| %>
+          <div  id="single-comment"> 
+            <span> <%= comment.author.name %> : <%= comment.text %></span>
```
Comment (elisha2kyakpo1) [inline app/views/posts/index.html.erb]: - [ ] Kindly make use of the callback function you created which should return the 5 most recent comments for a post.

**Review by elisha2kyakpo1 — CHANGES_REQUESTED** [review]
Hi 👋 @juanmanuelbyc, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✔️ The PR title and description look good and professional.
✔️ No linter errors.
✔️ Descriptive readme file.

## Required Changes ♻️
_Check the comments under the review._

- [ ] The license link gave me a 404 error when I clicked on it. We have to make sure our readme file links are working to make it look professional. Kindly fix the license link before resubmitting for another review.

## Optional suggestions

- [ ] Kindly add a root path to your app, the user's index action will be ideal for this app.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by afanimartin — APPROVED** [review]
Hi @juanmanuelbyc, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- Callback for 5 most recent comments used as requested 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

## Find me on slack
### Martin Afani

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**